image/gen/src/lib.rs

```diff
@@ -44,6 +44,9 @@ pub trait ImageGenratorExecutable {
 
 /// Image Gnerator Crypto Trait
 pub trait ImageGeneratorCrypto {
+    /// Calculate SHA-384 digest
+    fn sha256_digest(&self, data: &[u8]) -> anyhow::Result<[u32; 8]>;
```

Comment:
Can you make 8 a const in image/types and check that it doesn't change the ROM binary? If it does you can leave the const in this file.

Reply:
Done